Remaining section review (#11)
ezsystems/commerce-documentation: docs/api/business_api/baseoperation/loadproducts.md

## CatalogService

`CatalogService` provides a method for searching catalog elements using the proper data provider.
It also gets prices and total count of elements without offset or limit.

!!! note

>     `fetchChildrenList` returns an array with 2 elements:

Reviewer: ```suggestion
    `fetchChildrenList` returns an array with two elements:
```

## Configuration

> Filters `catalogList` and `productList` are defined in the configuration file.

Reviewer: ```suggestion
`catalogList` and `productList` filters are defined in the configuration file.
```

> They are used in catalog data provider to filter and sort all elements or products list from catalog.

Reviewer: ```suggestion
They are used in catalog data provider to filter and sort all elements or products lists from catalog.
```

> They are used in catalog data provider to filter and sort all elements or products list from catalog.

Author: ```suggestion
They are used in catalog data provider to filter and sort all elements or product lists from catalog.
```
